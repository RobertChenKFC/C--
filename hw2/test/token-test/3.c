../assignment-test/3.c