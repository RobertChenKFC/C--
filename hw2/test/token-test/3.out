IDENTIFIER
IDENTIFIER
IDENTIFIER
OP_SUB
CONST_INT
OP_ADD
CONST_FLOAT
MK_LPAREN
OP_MUL
CONST_FLOAT
IDENTIFIER
MK_RPAREN
OP_NOT
MK_LBRACE
[Error] Lexer: unrecognized token " on line 5
